code/node-server/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "node index.js",
     "test": "jest",
-    "watch": "supervisor -n index.js --no-restart-on error index.js"
+    "watch": "supervisor -n index.js --no-restart-on error --inspect index.js"
```

Comment:
Just wanted to point this out: `--inspect` allows you to use the chrome console to debug your server code. Drop debuggers in there and go HAM

![bitmoji](https://render.bitstrips.com/v2/cpanel/94f41e13-f754-4f02-8537-5489057fe24c-7c243825-53cb-4c20-a7d9-3fc8d332e86d-v1.png?transparent=1&palette=1&width=246)